algorithms/appo/tests/test_model.py

```diff
@@ -25,7 +25,9 @@ def test_forward_pass(self):
         torch.backends.cudnn.benchmark = True
 
         actor_critic = create_actor_critic(cfg, env.observation_space, env.action_space)
-        device = torch.device('cuda')
+        device_type = 'gpu' if torch.cuda.is_available() else 'cpu'
```

Comment:
Can you please instead make this a parameter of the test function, and then skip the GPU test with @unittest.skipUnless when CUDA is not available

Reply:
So do you mean to convert this into 2 different tests? and skip the GPU one if it's not available right?

Reply:
yes